Library/Homebrew/cmd/help.rb

```diff
@@ -89,4 +93,15 @@ def command_help(path)
       end.join.strip
     end
   end
+
+  def command_help_cmd(path)
+    path_str = path.to_s
+    cmd = path_str.match(%r{Homebrew/cmd/(.*).rb})[1]
```

Comment:
This can also be done with Ruby's built-in path functionality, which is a bit nicer than using a Regex:

```ruby
Pathname(path).basename(".rb")
```